assemblyline/odm/models/alert.py

```diff
@@ -165,41 +173,45 @@ def __hash__(self) -> int:
-    events = odm.sequence(odm.compound(Event), default=[], description="An audit of events applied to alert")
-    workflows_completed = odm.Boolean(default=False, description="Have all workflows ran on this alert?")
+    alert_id = odm.Keyword(copyto="__text__", description="Unique identifier for the alert.")
+    al = odm.compound(ALResults, description="Contains the results of the Assemblyline analysis for the alert.")
+    archive_ts = odm.Optional(odm.Date(description="Timestamp indicating when the alert was archived in the system."))
+    attack = odm.Compound(Attack, description="Structured data representing MITRE ATT&CK information associated with the alert.")
+    classification = odm.Classification(description="Security classification level of the alert.")
+    expiry_ts = odm.Optional(odm.Date(store=False), description="Timestamp indicating when the alert is scheduled to expire from the system.")
+    extended_scan = odm.Enum(values=EXTENDED_SCAN_VALUES, description="Indicates the status of an extended scan, if applicable. Extended scans are additional analyses performed after the initial analysis.")
+    file = odm.Compound(File, description="Information about the file associated with the alert.")
+    filtered = odm.Boolean(default=False, description="Indicates whether the alert results are filtered out based on system or user configurations.")
+    heuristic = odm.Compound(Heuristic, description="Data regarding the heuristics that triggered the alert.")
+    label = odm.List(odm.Keyword(), copyto="__text__", default=[], description="Labels assigned to the alert for categorization and filtering.")
+    metadata = odm.FlattenedObject(default={}, store=False, description="Additional metadata provided with the file at the time of submission.")
+    owner = odm.Optional(odm.Keyword(), description="The username or system component that owns or is responsible for the alert.")
+    priority = odm.Optional(odm.Enum(values=PRIORITIES), description="Indicates the importance level assigned to the alert.")
+    reporting_ts = odm.Date(description="Timestamp when the alert was created.")
+    submission_relations = odm.sequence(odm.compound(Relationship), description="Describes the hierarchical relationships between submissions that contributed to this alert.")
+    sid = odm.UUID(description="Identifier for the submission associated with this alert.")
+    status = odm.Optional(odm.Enum(values=STATUSES), description="Reflects the current state of the alert throughout its lifecycle. This status is subject to change as a result of user actions, automated processes, or the execution of workflows within Assemblyline. The status provides insight into the current phase of analysis or response.")
+    ts = odm.Date(description="Timestamp of when the file submission occurred that led to the generation of this alert.")
+    type = odm.Keyword(description="The type or category of the alert.")
```

Comment:
Can we elaborate a little more on `type`?  Seems like an important attribute.  Any possible values we can expect, or examples of values?

Reply:
I believe `type` in this case usually follows the `type` that's set at submission time, this is freeform and at the discretion of the submitter

Reply:
`USER` and `INGEST` seem very popular...